src/components/button/_button.scss

```diff
@@ -80,14 +81,26 @@
       border: 0;
     }
 
-    &:hover,
-    &:focus {
+    // Ordering of this pseudo selector is important to ensure
+    // It replaces hover state when element is focussed and mouse is hovering
+    &:hover {
+      color: govuk-colour("white");
```

Comment:
I believe that the white here is `$govuk-button-text-colour`, do we need to set it again?

I imagine if the focus styles were at the end we could avoid having to set the text colour again...